explainaboard/processors/conditional_generation.py

```diff
@@ -1,6 +1,5 @@
 from __future__ import annotations
```

Comment:
Could we revert the modifications to this file and save them for a future PR? I understand what you're doing here, but I'd like to discuss the design of this a bit more, but and don't want to hold up merging this PR. As far as I can tell this wouldn't be too harmful to the other parts.

Reply:
OK, I can do it and done.